Makefile

```diff
@@ -33,12 +33,16 @@ verify:
 
 .PHONY: build
 build:
-	$(GO_CMD) build -o bin/kwok ./cmd/controller/*.go
+	$(GO_CMD) build -o bin/kwok ./cmd/kwok/*.go
 
 .PHONY: integration-test
 integration-test:
 	@echo "Not implemented yet"
 
 .PHONY: e2e-test
-e2e-test:
-	@echo "Not implemented yet"
+e2e-test: \
```

Comment:
`\` is not needed, right?

Reply:
``` Makefile
.PHONY: e2e-test
e2e-test: \
	e2e-test-kwok \
	e2e-test-kwokctl

.PHONY: e2e-test-kwok
e2e-test-kwok:
	./test/kwok/kwok.test.sh

.PHONY: e2e-test-kwokctl
e2e-test-kwokctl:
	./test/kwok/kwokctl.test.sh
```

This is the final result I expect.
`\` is for the next line to follow.

Reply:
Or, just use one e2e entry, and local test can be run individually by shell argument?

```
e2e-test:
	./hack/e2e-test.sh
```

By default, `hack/e2e-test.sh` runs all e2e test. And we can run kwok / kwokctl by: `hack/e2e-test.sh kwok [or kwokctl]`.